src/main/gpiiConnector.js

```diff
@@ -770,7 +770,7 @@ fluid.defaults("gpii.app.dev.gpiiConnector.qss", {
         "http://registry\\.gpii\\.net/common/highContrastTheme": { value: "regular-contrast" },
         "http://registry\\.gpii\\.net/common/selfVoicing/enabled": { value: false },
         // use the initial value of the language as default setting
-        "http://registry\\.gpii\\.net/common/language": { value: "{systemLanguageListener}.model.configuredLanguage" }
+        "http://registry\\.gpii\\.net/common/language": { value: "{that}.options.defaultPreferences.systemDefaultLanguage" }
```

Comment:
Please add a comment here to list the JIRA number, and add a comment to indicate that this is a provisional fix related to understanding why the default value isn't taken from the core's logic (as per meeting discussion on 25/3/19) and that we need to circle back at some point to remove this special case